src/main/resources/add-ons-to-index.json

```diff
@@ -213,6 +222,9 @@
       "uid": "org.openmrs.module.appui",
       "type": "OMOD",
       "name": "App UI Module",
+	  "tags": [
+        "ui"
```

Comment:
"ui" tag is not helpful, it doesn't convey anything useful to the user.
You might use a tag like "back-end" for this, since it doesn't have any end-user-facing functionality.